dangvuthang/rocodingtest
# Feat/glance tracking

### Changelog

- Introduce **GlanceTracker** component and its dependency

### How to test

- **checkout** this branch
- `cd frontend`
- `npm install` then `npm run dev`
- Navigate to `localhost:3000/exam2`
- Check if looking outside of the screen cause alert message to show

### Limitation

- There is still some **false positives** caused by this model
